tests/radius_test.py

```diff
@@ -0,0 +1,122 @@
+import imp
+import os
+import sys
+import traceback
+import json
+from unittest import mock
+
+from click.testing import CliRunner
+
+import config.main as config
+import show.main as show
+
+test_path = os.path.dirname(os.path.abspath(__file__))
+modules_path = os.path.dirname(test_path)
+sys.path.insert(0, test_path)
+sys.path.insert(0, modules_path)
+
+import mock_tables.dbconnector
+
+show_radius_default_output="""\
+RADIUS global auth_type pap (default)
+RADIUS global retransmit 3 (default)
+RADIUS global timeout 5 (default)
+RADIUS global passkey <EMPTY_STRING> (default)
+
+"""
+
+config_radius_empty_output="""\
+"""
+
+config_radius_server_invalidkey_output="""\
+--key: Valid chars are ASCII printable except SPACE, '#', and ','
+"""
+
+config_radius_invalidipaddress_output="""\
+Invalid ip address
+"""
+
+class TestRadius(object):
+    @classmethod
+    def setup_class(cls):
+        os.environ['UTILITIES_UNIT_TESTING'] = "1"
+        print("SETUP")
+        import config.main
```

Comment:
does this check o/p of "show aaa" with login as radius

Reply:
It checks 'show radius' output. I could not find any existing 'show aaa' UT for TACACS+. Does a new test need to be added for 'show aaa' with RADIUS login?

Reply:
It will be nice, if it could be added. Can we ?

Reply:
Done.